WowUp.Common/WowUp.Common.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="protobuf-net" Version="3.0.29" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
```

Comment:
Can we remove this? I was able to build the PR without it
_Originally posted by @jliddev in https://github.com/jliddev/WowUp/pull/64#discussion_r496838681_

---
Not sure if others will have the issue as well. The package is not on my PC and I have the full visual studio install for dotnet/core/c#/wpf

Reply:
Oh, i see what you mean. not sure how that weird ref got in there. I removed the local ref and added this package to match in dev.